Module1 Homework/README.md

```diff
@@ -0,0 +1,184 @@
+> ### This README contains the answers to Module 1's homework questions.
+
+## Part 1 - Short-Answer Questions
+
+#### _a) What does the command `git status` output?_
+
+The `git status`  command shows the current state of the local Git repository. This includes the active branch and modified files, both staged and unstaged, as well as its position in relation to any connected remote repositories.
+
+#### _b) In SwiftUI, anything that gets drawn on the screen is a **`View`**._
+
+#### _c) print(“Hello world”) is an example of a **`(global) function`** call._
+#### _viewModel.getData() is an example of a **`method`** call._
+
+#### _d) Name some Views you have seen so far in SwiftUI._
+
+- `HStack`
+- `VStack`
+- `Text`
+- `Slider`
+
+
+#### _e) How do you create a new local repository using git? (Feel free to answer with how you use git, i.e. terminal or another app)_
+
+In Terminal (at the root folder of the project), execute:
+`git init` 
+
+#### _f) How do you preview your app in multiple orientations?_
+
+There are a number of ways to do this in Xcode:
+
+- Click on the `variants` button and select `orientation variants`
+
+- Create multiple previews using the `#Preview` macro in Xcode 15:
+```
+#Preview("Portrait") {
+    ContentView()
+        .previewInterfaceOrientation(.portrait)
+}
+#Preview("Landscape Left") {
+    ContentView()
+        .previewInterfaceOrientation(.landscapeLeft)
+}
```

Comment:
This won't work anymore in Xcode 15.
We can define the orientation like this:

```swift
#Preview(traits: .landscapeLeft) {
    ContentView()
}
```

We're going to need this in one of the next modules. :]

Reply:
Interestingly, `.previewInterfaceOrientation(.landscapeLeft)` still works for me in Xcode 15 with `#Preview` macro.  However, I had missed the whole `traits` parameter on the `#Preview` macro - and obviously the intended way to proceed moving forward.  Learned something there.  Thanks.